lib/screens/authentication/import_key/interactor/viewmodels/import_key_bloc.dart

```diff
@@ -45,6 +59,11 @@ class ImportKeyBloc extends Bloc<ImportKeyEvent, ImportKeyState> {
 
   void _findAccountFromWords(FindAccountFromWords event, Emitter<ImportKeyState> emit) {
     final authData = GenerateKeyFromRecoveryWordsUseCase().run(state.userEnteredWords.values.toList());
-    add(FindAccountByKey(privateKey: authData.eOSPrivateKey.toString(), words: authData.words));
+    final alternateAuthData = GenerateKeyFromSeedsPassportWordsUseCase().run(state.userEnteredWords.values.toList());
+    add(FindAccountByKey(
+      privateKey: authData.eOSPrivateKey.toString(),
+      alternatePrivateKey: alternateAuthData.eOSPrivateKey.toString(),
```

Comment:
Ok here you generate both types rigth and pass as event arguments

Reply:
Yes!